src/InstrumentationEngine/dllmain.cpp

```diff
@@ -27,11 +27,12 @@ BOOL APIENTRY DllMain(HINSTANCE hInstance, DWORD dwReason, LPVOID lpReserved)
             _CRTDBG_LEAK_CHECK_DF // dump leak report at shutdown
         );
 #endif
-        //INITIALIZE_REF_RECORDER(MicrosoftInstrumentationEngine::CRefCount::EnableRecorder);
+        INITIALIZE_REF_RECORDER(MicrosoftInstrumentationEngine::CRefCount::EnableRecorder);
```

Comment:
How bad is the impact in debug mode when you turn this on?

Reply:
It depends. It is pretty well eclipsed by having debug on at all. For example, one of my tests goes from running in 9 seconds in release to running in 2.5 minutes in debug. This adds another few seconds I think. But I could double check.